.idea/misc.xml

```diff
@@ -5,19 +5,23 @@
       <map>
```

Comment:
we should .gitignore this directory 

https://stackoverflow.com/questions/53147300/what-is-idea-folder-in-android-studio-3-2-12